tools/pkglint/lib/rules.ts

```diff
@@ -1689,3 +1745,17 @@ function cdkMajorVersion() {
   const releaseJson = require(`${__dirname}/../../../release.json`);
   return releaseJson.majorVersion as number;
 }
+
+/**
+ * Should this package be included in the monolithic package.
+ */
+function isIncludedInMonolith(pkg: PackageJson): boolean {
```

Comment:
@eladb I ended up duplicating this logic from `ubergen`. The logic in `ubergen` relies on the fact that the `ubergen.exclude` is not set on all modules that needs to excluded :/ . It's kinda of tricky, the **TL;DR** is that if the module is not `uber.exclude` there is a special logic that checks if it needs to be considered deprecated: [code](https://github.com/aws/aws-cdk/blob/master/tools/ubergen/bin/ubergen.ts#L104).